README.md

```diff
@@ -4,127 +4,132 @@
 
 ## Introduction
 
-This helm chart installs the [Scalyr Agent monitor](https://app.scalyr.com/help/scalyr-agent) into a Kubernetes
-cluster. It supports installing the agent with all features required to support a Kubernetes cluster monitoring.
-Additionally, it can deploy Scalyr agents which monitor other parts of the infrastructure (for example a hosted
-database service etc.).
+This helm chart installs the [Scalyr Agent](https://app.scalyr.com/help/scalyr-agent) in a Kubernetes
+cluster. Two Agent Plugins plugins are also installed:
+- The [Kubernetes monitor](https://app.scalyr.com/monitors/kubernetes) enables the Agent to monitor Kubernetes clusters.
+- The [Kubernetes Events monitor](https://app.scalyr.com/monitors/kubernetes-events) collects events from the Kubernetes API server for all nodes except master.
```

Comment:
Events monitor collects state change events from the whole Kubernetes cluster.

I assume you added "collects events from the Kubernetes API server for all nodes except master." because it has no schedule: master constraint, but that's primarily used to make sure it gets scheduled on some non-master node so we don't put additional load on the master node.

In short - Kubernetes Events Monitors is a singleton - this means only a single instance of it is active at the same time, but that single instance fetches events from the whole Kubernetes cluster. Monitor itself is scheduled on all the agent DaemonSet pods (expect on master node) and then internally this monitor uses leader election to pick out single pod on which it will run.